test/InvocationModel.test.ts

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License.
+
+import { CoreInvocationContext, RpcInvocationRequest, RpcParameterBinding } from '@azure/functions-core';
```

Comment:
The src vs test code is out of balance again so I would like you to remove this file. Last time I did not block the PR because usually people have the opposite problem (not enough testing), but this time I think I will. It's not an efficient use of your time to author and my time to review, so I don't want this to keep happening

We already have tests in the worker covering this scenario [here](https://github.com/Azure/azure-functions-nodejs-worker/blob/v3.x/test/eventHandlers/InvocationHandler.test.ts#L535). You _could_ add a test there to reflect your new flag, but honestly my preference is that you don't. Your src code changes amounted to a single boolean in one if-condition and the test itself would be more complicated than that

Reply:
Yeah I didn't realize those same tests existed on the worker repo. I created this file more as a way of testing the existing code (and providing a scaffolding for future `InvocationModel` tests that I thought didn't exist yet), than testing my own changes.

That said, if the code lives here, why are the tests in the worker repo? Those tests won't even make sense anymore if at some point we decide to switch the default framework to v4

Reply:
> That said, if the code lives here, why are the tests in the worker repo?

First of all, this whole library pattern is still a baby okay 😝 It will be a while before everything is put in the "right" place and you'll just have to look in both places for a while. Secondly, I wanted to keep a decent amount of testing in the worker to ensure the worker & library are integrating correctly. Eventually we will want some proper "integration" testing between the two, but that doesn't really exist yet. Lastly, I'm not worried about defaulting to v4 - that's likely years away.

Your heart is in the right place - I love when people improve the test/engineering infrastructure. It's just not sustainable if you try to do that as a part of every bug fix and feature. You have to do it separately and consciously choose which engineering work is most important. Just because you happen to touch this code, doesn't mean this code is most in need of test/engineering improvement.

Reply:
Ok, removed this test file